zellij-server/src/panes/terminal_pane.rs

```diff
@@ -285,6 +300,41 @@ impl Pane for TerminalPane {
     fn drain_messages_to_pty(&mut self) -> Vec<Vec<u8>> {
         self.grid.pending_messages_to_pty.drain(..).collect()
     }
+
+    fn start_selection(&mut self, start: &Position) {
+        self.grid.start_selection(start);
+        self.set_should_render(true);
+    }
+
+    fn update_selection(&mut self, to: &Position) {
+        // TODO: decide how often to scroll
+        let should_scroll = self.selection_scrolled_at.elapsed() >= time::Duration::from_millis(50);
```

Comment:
Maybe I'm misunderstanding, but if we have the `mio` timeout in the stdin thread, why do we need to keep track of the time here too?

Reply:
a bit hard to explain, but basically if we don't have it scrolling speed becomes faster when moving the mouse, as every change of position generates a new mouse hold event. This way it stays constant both while moving the cursor and when it's still, and the events are generated by the repeater